arbitrum-docs/bold/bold-gentle-introduction.md

```diff
@@ -0,0 +1,139 @@
+---
+title: 'A gentle introduction: BOLD'
+sidebar_label: 'A gentle introduction'
+description: 'An educational introduction that provides a high-level understanding of BOLD, a new dispute protocol to enable permissionless validation for Arbitrum chains.'
+user_story: 'As a user or researcher of the Arbitrum product suite, I want to learn about BOLD, a next-generation dispute protocol that enables permissionless validation.'
+content_type: gentle-introduction
+author: leeederek
+sme: leeederek
+---
+
+import PublicPreviewBannerPartial from './partials/_bold-public-preview-banner-partial.md';
+
+<PublicPreviewBannerPartial />
+
+This introduction is for those who want to learn about BOLD: a new dispute protocol for Optimistic Rollups that can enable **permissionless validation for Arbitrum chains**. BOLD stands for Bounded Liquidity Delay and is currently deployed on a public testnet for anyone to join and test how challenges will work. 
+
+This next-generation dispute protocol technology will soon be available for any Arbitrum chain, and pending a governance vote, will eventually be made available on Arbitrum Sepolia, Arbitrum One, and Arbitrum Nova.
+
+BOLD will eventually replace the current, permissioned fraud proof mechanism that powers Arbitrum chains today.
+
+## In a nutshell:
+
+- BOLD, an acronym for Bounded Liquidity Delay, is a new challenge protocol for Arbitrum chains that ensures disputes will take, at most, 1 challenge period (6.4 days) to get resolved. BOLD effectively eliminates delay attacks by enforcing a upper time-bound for dispute resolution.
```

Comment:
I think we should mention that the upper time-bound enforced for resolving disputes is added to the current challenge period (e.g., if 3 days have passed since the assertion was posted, and someone initiates a challenge, the full confirmation period will be 10 days)

Reading the text I assumed that I can withdraw my funds after 1 challenge period regardless of whether or not disputes arise, and that's not the case, and that might be a bit confusing.